src/reducers/aboutUs/aboutUsReducer.js

```diff
@@ -0,0 +1,31 @@
+import initialState from "../../store/initialState";
+import {
+  GET_MEMBERS_PENDING,
+  GET_MEMBERS_SUCCESS,
+  GET_MEMBERS_ERROR,
+} from "../../actions/aboutUs/membersActions";
+
+const aboutUsReducer = (state = initialState.aboutUs, action) => {
```

Comment:
Keep initial state for each reducer, near by the reducer

Reply:
done